Formula/k/ktx.rb

```diff
@@ -0,0 +1,44 @@
+class Ktx < Formula
+  desc "Khronos Texture Library and Tools"
+  homepage "https://github.com/KhronosGroup/KTX-Software"
+  url "https://github.com/KhronosGroup/KTX-Software/archive/refs/tags/v4.3.2.tar.gz"
+  sha256 "74a114f465442832152e955a2094274b446c7b2427c77b1964c85c173a52ea1f"
+  license all_of: [
+    "Apache-2.0",
+    "BSD-1-Clause",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "BSL-1.0",
+    "CC-BY-3.0",
+    "CC-BY-4.0",
+    "CC0-1.0",
+    "MIT",
+    "Zlib",
+    :cannot_represent, # Cesium Trademark Terms, ETCSLA, HI Trademark, Kodak, PNGSuite
```

Comment:
This sounds like it might not be open source. Are there other package managers shipping this software?

Reply:
Yes, although a cursory search only found one:
https://www.freshports.org/graphics/khronos-texture/

The licenses listed are taken from the files in the repository; glancing over them didn’t reveal anything specifically closed-source, but IANAL.

Reply:
Let's make the whole thing `:cannot_represent` it seems better than this whole list of things that may or may not apply

Reply:
What should I put in the comment? Just “see this repo URL”?

Reply:
Something like this?
> Too complicated to represent in the formula